vllm/sampling_params.py

```diff
@@ -244,7 +244,7 @@ def from_optional(
     ) -> "SamplingParams":
         if logit_bias is not None:
             logit_bias = {
-                int(token): bias
+                int(token): min(100.0, max(-100.0, bias))
```

Comment:
nit: Could you add the original comment about why we clamp this value? 😅

Reply:
Sure, just added.